CONTRIBUTING.md

```diff
@@ -49,6 +55,33 @@ All changes must be code reviewed. Coding conventions and standards are explaine
 docs](https://github.com/kubernetes/community/tree/master/contributors/devel). Expect reviewers to request that you
 avoid common [go style mistakes](https://github.com/golang/go/wiki/CodeReviewComments) in your PRs.
 
+## Reviewing a Patch
+
+### Approvals
+
+Please see the [Kubernetes community document on pull
+requests](https://git.k8s.io/community/contributors/guide/pull-requests.md) for more information about the merge
+process.
+
+- A PR is approved by one of the project maintainers and owners after reviews.
+- Approvals should be the very last action a maintainer takes on a pull request.
```

Comment:
I know this is a topic subject of many discussions in Kubernetes land...
IMO It will be great to disable the bot that automatically applies /approve for maintainers, so all the PRs can follow the same approach and every PRs requires approval from someone different than the author

Reply:
Ohhh... I see now https://github.com/kubernetes-sigs/cluster-api/pull/3191#issuecomment-644356312
I'm happy to help with this if there is agreement

Reply:
That is a fine change, although I want to call out that even in that case, the above guideline stands.